src/index.ts

```diff
@@ -95,10 +95,29 @@ module.exports = class ServerlessPlugin {
 
   private async afterPackageFunction() {
     const config = getConfig(this.serverless.service);
+    const forwarderArn: string | undefined = config.forwarderArn;
+    const forwarder: string | undefined = config.forwarder;
+
+    let datadogForwarderArn;
     if (config.enabled === false) return;
-    if (config.forwarder) {
+
+    if (forwarderArn && forwarder) {
+      throw new Error(
+        "Error: Both 'forwarderArn' and 'forwarder' parameters are set. Please only use the 'forwarderArn' parameter.",
+      );
+    } else if (forwarderArn !== undefined && forwarder === undefined) {
+      datadogForwarderArn = forwarderArn;
+    } else if (forwarder !== undefined && forwarderArn === undefined) {
+      datadogForwarderArn = forwarder;
+    } else {
+      throw new Error(
```

Comment:
Does this mean the `forwarder`/`forwarderArn` is no longer optional? We want to still be able to deploy without these parameters (we just won't create subscription filters)

Reply:
Deploying without forwarder/forwarderArn is still optionally, a warning message appears but deployment still continues